From c5f3d0e4e43acc8339df307c09ed1a212979677d Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 19 Jan 2018 20:32:02 -0700
Subject: [PATCH 1/6] klte-common: power: Add legacy qcom HAL compat code

* msm8974-common may end up going back and forth between legacy
  and HIDL HALs, so let's be compatible with both for the time
  being.

Change-Id: I5067fc474d7878a0f68e25c70b723de0e813d7c1
---
 power/power_ext.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/power/power_ext.c b/power/power_ext.c
index 0506b1c..e7289c6 100644
--- a/power/power_ext.c
+++ b/power/power_ext.c
@@ -52,3 +52,7 @@ void power_set_interactive_ext(int on) {
     sysfs_write(TK_POWER, on ? "1" : "0");
     sysfs_write(TS_POWER, on ? "1" : "0");
 }
+
+void cm_power_set_interactive_ext(int on) {
+    power_set_interactive_ext(on);
+}
-- 
2.7.4

